modules/ROOT/pages/security/configuration.adoc

```diff
@@ -1,16 +1,24 @@
 = Configuration
 :description: This page describes how to configure authentication and authorization features in the Neo4j GraphQL Library.
 
+The Neo4j GraphQL Library uses JSON Web Token (JWT) authentication.
+JWTs are tokens containing claims or statements about the user or client making the request.
+These claims can include information such as the user's ID or roles.
+
+A JWT can be obtained from an authentication service and is then included in an API request.
+The API verifies the JWT and returns the requested data if the JWT is valid.
+
 == Instantiation
 
-The Neo4j GraphQL Library can accept JSON Web Tokens via two mechanisms:
+The Neo4j GraphQL Library can accept two types of JWTs:
 
 * Encoded JWTs in the `token` field of the request context.
 * Decoded JWTs in the `jwt` field of the request context.
 
-If using the former, the library will need to be configured with a key to decode and verify the token.
+=== Encoded JWTs
 
-The following code block demonstrates using Apollo Server, extracting the `Authorization` header from the request, and putting it into the appropriate context field:
+To use encoded JWTs, the library must to be configured with a key to decode and verify the tokens.
```

Comment:
This isn't the case.

An encoded token can be decoded via a JWKS endpoint.

Reply:
rephrased